scaaml/capture/input_generators/attack_point_iterator.py

```diff
@@ -152,3 +153,29 @@ def __iter__(self):
 
     def get_generated_keys(self) -> List[str]:
         return [self._name]
+
+
+class AttackPointIteratorRepeat(AttackPointIterator):
+    """
+    Attack point iterator class that iterates 
+    over a configuration a repeated amount of times.
+    """
+
+    def __init__(self, operation: str, repetitions: int,
+                 configuration: Dict) -> None:
+        """Initialize the repeated iterate."""
```

Comment:
Documentation what are possible values of arguments. Especially for `repetitions` what happens when that is 0 or negative? Is there a way to repeat forever? Unittests missing. Check with the design doc.